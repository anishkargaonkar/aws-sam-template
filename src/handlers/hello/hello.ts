import { APIGatewayProxyResult } from 'aws-lambda';
import logger from '../utils/logger';

export const handler = async (): Promise<APIGatewayProxyResult> => {
	logger.info(`time to say 'hello'`);
  return { body: JSON.stringify({ message: 'hello' }), statusCode: 200 };
};
