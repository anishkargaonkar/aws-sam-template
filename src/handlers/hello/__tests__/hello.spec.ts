import { handler } from "../hello";

describe('hello handler', () => {
  it('should say hello', async () => {
    const result = await handler();
    expect(result).toMatchSnapshot();
  });
});
