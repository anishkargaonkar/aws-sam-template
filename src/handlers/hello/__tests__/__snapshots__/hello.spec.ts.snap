// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hello handler should say hello 1`] = `
Object {
  "body": "{\\"message\\":\\"hello\\"}",
  "statusCode": 200,
}
`;
